app/App/Account/Account/Balances/Balance/index.js

```diff
@@ -30,7 +30,7 @@ class Balance extends React.Component {
         priceChangeClass += ' signerBalanceCurrentPriceChangeDown'
       }
     }
-    let name = balance.name
+    let name = balance.name || ''
```

Comment:
which can this be undefined?

Reply:
We need to add a migration to set the names for the native currencies, the initial state now has them all set.
I can make that migration now if you'd like?

This is causing crashes in develop atm when expanding balances, Sam has posted a screenshot in the notion doc!